<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Servant_open</class>
 <widget class="QWidget" name="Servant_open">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Servant_open</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;WELCOME!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;WORK&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;PRIVATE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
                background-color:rgba(100,225,100,30);//背景色（也可以设置图片）
  border-style:outset; //边框样式（inset/outset）
  border-width:4px;  //边框宽度像素
border-radius:10px;        //边框圆角半径像素
border-color:rgba(255,255,255,30);    //边框padding:6px;                //填衬
                }
                //鼠标按下样式
                QPushButton:pressed{
                background-color:rgba(100,255,100,200);
                border-color:rgba(255,255,255,30);
                border-style:inset;
                color:rgba(0,0,0,100);
                }
                //鼠标悬停样式
                QPushButton:hover{
                background-color:rgba(100,255,100,100);
                border-color:rgba(255,255,255,200);
                color:rgba(0,0,0,200);
                }</string>
   </property>
   <property name="text">
    <string>点单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
                background-color:rgba(100,225,100,30);//背景色（也可以设置图片）
  border-style:outset; //边框样式（inset/outset）
  border-width:4px;  //边框宽度像素
border-radius:10px;        //边框圆角半径像素
border-color:rgba(255,255,255,30);    //边框padding:6px;                //填衬
                }
                //鼠标按下样式
                QPushButton:pressed{
                background-color:rgba(100,255,100,200);
                border-color:rgba(255,255,255,30);
                border-style:inset;
                color:rgba(0,0,0,100);
                }
                //鼠标悬停样式
                QPushButton:hover{
                background-color:rgba(100,255,100,100);
                border-color:rgba(255,255,255,200);
                color:rgba(0,0,0,200);
                }</string>
   </property>
   <property name="text">
    <string>查询桌位信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>190</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
                background-color:rgba(100,225,100,30);//背景色（也可以设置图片）
  border-style:outset; //边框样式（inset/outset）
  border-width:4px;  //边框宽度像素
border-radius:10px;        //边框圆角半径像素
border-color:rgba(255,255,255,30);    //边框padding:6px;                //填衬
                }
                //鼠标按下样式
                QPushButton:pressed{
                background-color:rgba(100,255,100,200);
                border-color:rgba(255,255,255,30);
                border-style:inset;
                color:rgba(0,0,0,100);
                }
                //鼠标悬停样式
                QPushButton:hover{
                background-color:rgba(100,255,100,100);
                border-color:rgba(255,255,255,200);
                color:rgba(0,0,0,200);
                }</string>
   </property>
   <property name="text">
    <string>查看评论</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
                background-color:rgba(100,225,100,30);//背景色（也可以设置图片）
  border-style:outset; //边框样式（inset/outset）
  border-width:4px;  //边框宽度像素
border-radius:10px;        //边框圆角半径像素
border-color:rgba(255,255,255,30);    //边框padding:6px;                //填衬
                }
                //鼠标按下样式
                QPushButton:pressed{
                background-color:rgba(100,255,100,200);
                border-color:rgba(255,255,255,30);
                border-style:inset;
                color:rgba(0,0,0,100);
                }
                //鼠标悬停样式
                QPushButton:hover{
                background-color:rgba(100,255,100,100);
                border-color:rgba(255,255,255,200);
                color:rgba(0,0,0,200);
                }</string>
   </property>
   <property name="text">
    <string>查看薪水</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>340</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
                background-color:rgba(100,225,100,30);//背景色（也可以设置图片）
  border-style:outset; //边框样式（inset/outset）
  border-width:4px;  //边框宽度像素
border-radius:10px;        //边框圆角半径像素
border-color:rgba(255,255,255,30);    //边框padding:6px;                //填衬
                }
                //鼠标按下样式
                QPushButton:pressed{
                background-color:rgba(100,255,100,200);
                border-color:rgba(255,255,255,30);
                border-style:inset;
                color:rgba(0,0,0,100);
                }
                //鼠标悬停样式
                QPushButton:hover{
                background-color:rgba(100,255,100,100);
                border-color:rgba(255,255,255,200);
                color:rgba(0,0,0,200);
                }</string>
   </property>
   <property name="text">
    <string>申请休假</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>440</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
                background-color:rgba(100,225,100,30);//背景色（也可以设置图片）
  border-style:outset; //边框样式（inset/outset）
  border-width:4px;  //边框宽度像素
border-radius:10px;        //边框圆角半径像素
border-color:rgba(255,255,255,30);    //边框padding:6px;                //填衬
                }
                //鼠标按下样式
                QPushButton:pressed{
                background-color:rgba(100,255,100,200);
                border-color:rgba(255,255,255,30);
                border-style:inset;
                color:rgba(0,0,0,100);
                }
                //鼠标悬停样式
                QPushButton:hover{
                background-color:rgba(100,255,100,100);
                border-color:rgba(255,255,255,200);
                color:rgba(0,0,0,200);
                }</string>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>440</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
                background-color:rgba(100,225,100,30);//背景色（也可以设置图片）
  border-style:outset; //边框样式（inset/outset）
  border-width:4px;  //边框宽度像素
border-radius:10px;        //边框圆角半径像素
border-color:rgba(255,255,255,30);    //边框padding:6px;                //填衬
                }
                //鼠标按下样式
                QPushButton:pressed{
                background-color:rgba(100,255,100,200);
                border-color:rgba(255,255,255,30);
                border-style:inset;
                color:rgba(0,0,0,100);
                }
                //鼠标悬停样式
                QPushButton:hover{
                background-color:rgba(100,255,100,100);
                border-color:rgba(255,255,255,200);
                color:rgba(0,0,0,200);
                }</string>
   </property>
   <property name="text">
    <string>上下班打卡</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>634</x>
     <y>502</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton{
                background-color:rgba(100,225,100,30);//背景色（也可以设置图片）
  border-style:outset; //边框样式（inset/outset）
  border-width:4px;  //边框宽度像素
border-radius:10px;        //边框圆角半径像素
border-color:rgba(255,255,255,30);    //边框padding:6px;                //填衬
                }
                //鼠标按下样式
                QPushButton:pressed{
                background-color:rgba(100,255,100,200);
                border-color:rgba(255,255,255,30);
                border-style:inset;
                color:rgba(0,0,0,100);
                }
                //鼠标悬停样式
                QPushButton:hover{
                background-color:rgba(100,255,100,100);
                border-color:rgba(255,255,255,200);
                color:rgba(0,0,0,200);
                }</string>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
